mon-pix/tests/unit/components/reset-password-form-test.js

```diff
@@ -77,66 +77,58 @@ describe('Unit | Component | reset password form', function() {
     });
 
     describe('When user password is saved', () => {
-      it('should update validation with success data', function() {
+      it('should update validation with success data', async function() {
         // given
-        component.set('user', userWithGoodPassword);
+        component = createGlimmerComponent('component:reset-password-form', { user: userWithGoodPassword });
 
         // when
-        run(() => {
-          component.send('handleResetPassword');
-        });
+        await component.handleResetPassword();
 
         // then
-        expect(component.get('validation')).to.eql(SUBMISSION_MAP['default']);
+        expect(component.validation).to.eql(SUBMISSION_MAP['default']);
       });
 
-      it('should update _displaySuccessMessage', function() {
+      it('should update _displaySuccessMessage', async function() {
         // given
-        component.set('user', userWithGoodPassword);
+        component = createGlimmerComponent('component:reset-password-form', { user: userWithGoodPassword });
 
         // when
-        run(() => {
-          component.send('handleResetPassword');
-        });
+        await component.handleResetPassword();
 
         // then
-        expect(component.get('_displaySuccessMessage')).to.eql(true);
+        expect(component.displaySuccessMessage).to.eql(true);
       });
 
-      it('should reset paswword input', function() {
+      it('should reset paswword input', async function() {
         // given
-        component.set('user', userWithGoodPassword);
+        component = createGlimmerComponent('component:reset-password-form', { user: userWithGoodPassword });
 
         // when
-        run(() => {
-          component.send('handleResetPassword');
-        });
+        await component.handleResetPassword();
 
         // then
-        expect(component.get('user.password')).to.eql(null);
+        expect(component.args.user.password).to.eql(null);
```

Comment:
Il me semble que c'est 
```suggestion
        expect(component.user.password).to.eql(null);
```

Reply:
Non car le `user` est passé en paramètres.

Reply:
Effectivement ! Mais c'est pas curieux de tester l'entrée en sortie (en quelque sorte) ?

Reply:
C'est vrai que ça peut être curieux, mais l'entrée est mutée par le composant et ce teste vérifie la mutation.
On a glimmeriser sans remettre ce principe en cause.